packages/andive/src/components/alert.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactChild } from 'react'
+import styled from 'styled-components'
+
+import Box from './box'
+import * as palette from '../constants/palette'
+
+const textColors = {
+  success: palette.successText,
+  error: palette.errorText,
+  warning: palette.warningText,
+  info: palette.darkBeetrootPurple,
+  disabled: palette.darkPrimary,
+}
+
+const backgroundColors = {
+  success: palette.lightLettuceGreen,
+  error: palette.lightRadishRed,
+  warning: palette.lightPotatoYellow,
+  info: palette.hexToRGBA(palette.lightBeetrootPurple, 0.6),
+  disabled: palette.border,
+}
+
+const AlertRoot = styled(Box)<{ variant: string, inCard: boolean }>`
+  padding: 8px, 16px, 8px, 16px;
+  background-color: ${props => backgroundColors[props.variant] || backgroundColors['warning']};
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+
+  border-top-left-radius: ${props => props.inCard ? 15 : 4}px;
+  border-top-right-radius: ${props => props.inCard ? 15 : 4}px;
+  border-bottom-right-radius: ${props => props.inCard ? 0 : 4}px;
+  border-bottom-left-radius: ${props => props.inCard ? 0 : 4}px;
+
+  & [data-andive-type='typography'] {
+    color: ${props => textColors[props.variant] || textColors['warning']};
```

Comment:
I would prefer the `warning` fallback to be in the React component, not the styled css.